Docs: small change  about addon promote rules (#456)
kubevela/catalog: README.md

- A new addon added in this repo should be put in as an experimental one unless you have test for a long time in your product environment and be approved by most maintainers.

> - An experimental addon must meet these conditions can be promoted as a verified one.

Reviewer: ```suggestion
- An experimental addon must meet these conditions to be promoted as a verified one.
```

  - This addon must be tested by addon's [e2e-test](./test/e2e-test/addon-test) to guarantee this addon can ben enabled successfully.

  - This addon must have some basic but necessary information.

    - An accessible icon url and source url defined in addon's `metadata.yaml`.
    
    - A detail introduction include a basic example about how to use and what's the benefit of this addon in `README.md`.
      
    - Also provide an introduction in KubeVela [documentation](https://kubevela.net/docs/reference/addons/overview).
    

>     - Better provide more useful examples in example [dir](examples/).

Reviewer: ```suggestion
    - It's more likely to be accepted if useful examples in example [dir](examples/) provided.
```

>     - It's more likely to be accepted if useful examples provided in example [dir](examples/).

Reviewer 2: ```suggestion
    - It's more likely to be accepted if useful examples are provided in example [dir](examples/).
```
